common/src/main/java/com/wynntils/functions/CombatFunctions.java

```diff
@@ -52,4 +52,64 @@ protected List<String> getAliases() {
             return List.of("agl", "above_ground_level");
         }
     }
+
+    public static class LastSpellNameFunction extends Function<String> {
+        @Override
+        public String getValue(FunctionArguments arguments) {
+            return arguments.getArgument("burst").getBooleanValue()
+                    ? Models.Spell.getLastBurstSpellName()
+                    : Models.Spell.getLastSpellName();
+        }
+
+        @Override
+        public FunctionArguments.Builder getArgumentsBuilder() {
+            return new FunctionArguments.OptionalArgumentBuilder(
+                    List.of(new FunctionArguments.Argument<>("burst", Boolean.class, true)));
```

Comment:
false is probably a better default, isn't it?

Reply:
Not sure. I don't really see a use case for non-burst other than just extra information, but the burst one is actually used to time recasts or cycles or whatever. If you really want to though, I can do false

Reply:
Unless you meant to invert the boolean

Reply:
I think the function name itself is more logical with burst mode off?

Reply:
Ah yea true